WordPress/WordPressUITests/LoginTests.swift

```diff
@@ -55,6 +53,6 @@ class LoginTests: XCTestCase {
 //        let email = WordPressTestCredentials.nuxEmailPrefix + username + WordPressTestCredentials.nuxEmailSuffix
 //
 //        createAccount(email: email, username: username, password: WordPressTestCredentials.oneStepPassword)
-//        waitForElementToAppear(element: app.tabBars[ elementStringIDs.mainNavigationBar ], timeout: 20)
+//        waitForElementToAppear(element: XCUIApplication().tabBars[ elementStringIDs.mainNavigationBar ], timeout: 20)
```

Comment:
Do we need this commented out code? Why keep it?

Reply:
These tests are needed to be re-implemented. I've moved them into separate issue: #8356